iap/example_gce_backend.py

```diff
@@ -0,0 +1,30 @@
+CLOUD_PROJECT_ID = 'YOUR_PROJECT_ID'
+BACKEND_SERVICE_ID = 'YOUR_BACKEND_SERVICE_ID'
+
+from flask import Flask
```

Comment:
Import modules, not members, so just `import flask` and use `flask.Flask` and `flask.request`.